tests/Workflow.py

```diff
@@ -81,3 +81,39 @@ def testGetRunsWithStrings(self):
             lambda r: r.id,
             [109950033, 108817672, 108794468, 107927403, 105213061, 105212023],
         )
+
+    def testCreateDispatchWithBranch(self):
+        dispatch_inputs = {"logLevel": "Warning", "message": "Log Message"}
+        workflow = self.g.get_repo("wrecker/PyGithub").get_workflow(
+            "manual_dispatch.yml"
+        )
+        branch = self.g.get_repo("wrecker/PyGithub").get_branch(
+            "workflow_dispatch_branch"
+        )
+
+        self.assertTrue(workflow.create_dispatch(branch, dispatch_inputs))
+
+    def testCreateDispatchWithTag(self):
+        dispatch_inputs = {"logLevel": "Warning", "message": "Log Message"}
+        workflow = self.g.get_repo("wrecker/PyGithub").get_workflow(
+            "manual_dispatch.yml"
+        )
+        tags = self.g.get_repo("wrecker/PyGithub").get_tags()
+        for tag in tags:
```

Comment:
tag = [t for t in tags if t.name == "workflow_dispatch_tag"]

Reply:
Done. Though I will add this form makes the API go and fetch all pages of tags from the repo. Its not that relevant here because we are using the recorded data file.